src/holofuel/UpdateHolofuelUserPrompt.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect, useCallback } from 'react'
+import { useHistory, Link } from 'react-router-dom'
+import { useQuery } from '@apollo/react-hooks'
+import { isEmpty } from 'lodash/fp'
+import HolofuelUserQuery from 'graphql/HolofuelUserQuery.gql'
+import useFlashMessageContext from 'holofuel/contexts/useFlashMessageContext'
+import { PROFILE_PATH } from 'holofuel/utils/urls'
+import 'holofuel/global-styles/colors.css'
+import 'holofuel/global-styles/index.css'
+
+function UserPromptMessage ({ newMessage }) {
+  const handleResetFlashMessage = useCallback(
+    () => newMessage('', 0),
+    [newMessage])
+
+  const userMessage = `It looks like you don't yet have a Holofuel Account Nickname. Visit your profile page to personalize your nickname so your peers can easily recognize you.`
+  const linkMessage = `Click here to visit your profile.`
+  return <>
+    <div className='message'>{userMessage}</div>
+    <br />
+    <Link to={PROFILE_PATH} onClick={handleResetFlashMessage}>{linkMessage}</Link>
+  </>
+}
+
+function UpdateHolofuelUserPrompt ({
```

Comment:
Still like `PromptForNickname`

I like using a verb to name these non rendering components, because they are DOING something, not BEING something. Eg `AcceptRequestOffers`